src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs

```diff
@@ -32,7 +39,21 @@ public static partial class FrameworkInfo
 
             if (clrVersion == 4)
             {
-                var release = Get45PlusLatestInstallationFromRegistry();
+                int? release = null;
+                try
+                {
+                    if (registryLock)
+                    {
+                        RegistryLocked = true;
+                    }
+                    release = Get45PlusLatestInstallationFromRegistry(RegistryLocked);
+                }
+                catch (Exception ex)
+                {
+                    _ = ex;
```

Comment:
What are we achieving with this?

Reply:
Nothing 😅, I'll remove the _ = ex; 
Yet, I wonder if we should Log that the Registry access failed or not.

Reply:
if you're not using the exception remove the whole `(Exception ex`) from the catch statement

Reply:
> Yet, I wonder if we should Log that the Registry access failed or not.

I think it would be nice. We can inject an instance of `SentryOptions` inside the event processor through constructor and then inject it here, I think?